searx/static/themes/simple/src/less/style.less

```diff
@@ -460,9 +477,7 @@ article[data-vim-selected].category-social {
   span.title,
   span.source {
     display: block;
-    position: absolute;
 
-    width: 100%;
     font-size: 0.9rem;
```

Comment:
Wow, looks quite pretty :rocket:   .. but there is an issue in the layout:

When we remove these properties from the `span.title` selector the result thumbnail is stretched by the title text ..

![image](https://github.com/searxng/searxng/assets/554536/a8d08415-50a6-44a7-9157-8151c2a87b7f)

before this patch it looks like this:

![image](https://github.com/searxng/searxng/assets/554536/20466f0c-afd8-4585-8450-847ca74647df)

Reply:
Without these changes, the texts didn't show at all, I'm not really sure what's causing this here :shrug: